README.md

```diff
@@ -16,78 +17,143 @@ This project aims to build the official site for the community page so that any
 
 ## Tech Stack 🗃
 
- <img src="https://img.shields.io/badge/-MongoDB-yellow?style=flat&logo=mongoDB"> <img src="https://img.shields.io/badge/-ExpressJS-grey?style=flat&logo=express&logoColor=white"> <img src="https://img.shields.io/badge/ReactJS%20-%2320232a.svg?logo=react" >   <img src="https://img.shields.io/badge/-NodeJS%20-%2320232a?style=flat&logo=node.js"> 
- 
+<img src="https://img.shields.io/badge/-MongoDB-yellow?style=flat&logo=mongoDB"> <img src="https://img.shields.io/badge/-ExpressJS-grey?style=flat&logo=express&logoColor=white"> <img src="https://img.shields.io/badge/ReactJS%20-%2320232a.svg?logo=react" > <img src="https://img.shields.io/badge/-NodeJS%20-%2320232a?style=flat&logo=node.js">
 
 ## How to contribute ? 🤔
 
 - Get started with the Contributon Guidelines mentioned [here](https://github.com/HITK-TECH-Community/Community-Website/blob/main/CONTRIBUTING.md)
 - Make sure to follow our conventions mentioned [here](https://github.com/HITK-TECH-Community/Community-Website/blob/main/frontend/conventions.md)
 
-
-##  Getting Started 👨‍💻
-
-**NOTE - You must be on NODE Version 14.19.0**
```

Comment:
can we please retain this line, also you can emphasise that this version is required to run this project

Reply:
Ok sure!